test/Microsoft.Azure.SignalR.Tests.Common/Microsoft.Azure.SignalR.Tests.Common.csproj

```diff
@@ -15,5 +15,6 @@
     <PackageReference Include="Microsoft.AspNetCore.Testing" Version="$(MicrosoftAspNetCoreTestingVersion)" />
     <PackageReference Include="Microsoft.Extensions.Configuration.UserSecrets" Version="$(MicrosoftConfigurationUserSecretVersion)" />
     <PackageReference Include="Microsoft.Extensions.Logging.Testing" Version="$(MicrosoftExtensionsLoggingPackageVersion)" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.3" />
```

Comment:
Why we need it? BTW, versions should be defined in `dependencies.props`

Reply:
I moved a util class to a more underlayer project. And we need to add reference to let this project to use ValueTask